Plan/src/main/java/com/djrapitops/plan/system/database/VelocityDBSystem.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * License is provided in the jar as LICENSE also here:
+ * https://github.com/Rsl1122/Plan-PlayerAnalytics/blob/master/Plan/src/main/resources/LICENSE
+ */
+package com.djrapitops.plan.system.database;
+
+import com.djrapitops.plan.api.exceptions.database.DBInitException;
+import com.djrapitops.plan.system.database.databases.sql.MySQLDB;
+import com.djrapitops.plan.system.locale.Locale;
+
+import java.util.function.Supplier;
+
+/**
+ * Velocity Database system that initializes MySQL object.
+ *
+ * Based on BungeeDBSystem
+ *
+ * @author MicleBrick
+ */
+public class VelocityDBSystem extends DBSystem {
```

Comment:
Duplication of this class is unnecessary, as only benefit is having it in velocity specific class.
This can be sorted later by refactoring common classes into network specific names